cmd/shipper-app/main.go

```diff
@@ -404,9 +404,10 @@ func startTrafficController(cfg *cfg) (bool, error) {
 	}
 
 	c := traffic.NewController(
-		client.NewShipperClientOrDie(traffic.AgentName, cfg.restCfg),
+		client.NewKubeClientOrDie(capacity.AgentName, cfg.restCfg),
+		cfg.kubeInformerFactory,
+		client.NewShipperClientOrDie(capacity.AgentName, cfg.restCfg),
```

Comment:
I'm a bit confused. If this is `startTrafficController`, why would it create a client the capacity agent name?

Reply:
Whoops, I guess? :)